packages/status-react/src/components/main-sidebar/components/chats/chat-group.tsx

```diff
@@ -12,7 +12,7 @@ interface Props {
   children: React.ReactNode
 }
 
-export const ChannelGroup = (props: Props) => {
+export const ChatGroup = (props: Props) => {
```

Comment:
non-blocking: is it just me or the group appears bold on the screenshot, expected?

Reply:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/13265126/182211694-24acc632-42b6-46bd-8f3e-6888ff80c1cc.png">

not bold, but rather diff weight, I guess